### DISCUSSION TITLE
Is the Prusa Mini supported?

### DISCUSSION THREAD

**Discussion by needmoredeeps** [feelfreelinux/octo4a#37] [Q&A] [answered]
Everything works great but for some reason it's not connecting to the Prusa mini. My phone says  the Prusa mini is plugged in and the app opens when it's plugged in but it does not connect. 

PRUSA mini

![Screenshot_20210923-201904](https://user-images.githubusercontent.com/91215822/134613109-4ebb7ce9-51cd-4663-940f-eca90740d076.png)

**Comment by feelfreelinux** [comment, marked as answer]
Will be supported in 1.0.0 which release is planned for tomorrow.

**Reply by needmoredeeps** [reply]
Thanks. Awesome work I'm starting to brush up on Android to see if I can help. I love this I got a few old phones.

**Reply by feelfreelinux** [reply]
1.0.0 was released yesterday. Please try that one :)
